eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -11,13 +11,21 @@ jobs:
           echo "##vso[build.addbuildtag]Scheduled"
         displayName: "Tag scheduled builds"
         condition: and(eq(variables['Build.SourceBranchName'],'master'),eq(variables['Build.Reason'],'Schedule'))
+      - pwsh: |
+          $skipDevBuildNumber = "false"
+          if ('$(Build.Reason)' -eq 'Manual') {
+            $skipDevBuildNumber = "true"
+          }
+          $versioningProperties = "/p:OfficialBuildId=$(OfficialBuildId) /p:SkipDevBuildNumber=$skipDevBuildNumber";
+          echo "##vso[task.setvariable variable=VersioningProperties]$versioningProperties"
+        displayName: "Setup Versioning Properties"
       - task: DotNetCoreInstaller@2
         displayName: "Use .NET Core sdk $(DotNetCoreSDKVersion)"
         inputs:
           version: "$(DotNetCoreSDKVersion)"
       - script: >-
           dotnet pack eng/service.proj -o $(Build.ArtifactStagingDirectory) -warnaserror /p:ServiceDirectory=${{parameters.ServiceDirectory}}
-          /p:PublicSign=false ${{ parameters.VersioningProperties }} /p:Configuration=$(BuildConfiguration) /p:CommitSHA=$(Build.SourceVersion)
+          /p:PublicSign=false $(VersioningProperties) /p:Configuration=$(BuildConfiguration) /p:CommitSHA=$(Build.SourceVersion)
```

Comment:
Would the $versioningProperties variable persist from the previous PowerShell step?